docker-compose.yml

```diff
@@ -68,3 +80,21 @@ services:
       - postgres
     links:
       - postgres
+
+  # Use this container to initialise a default user / organisation / project in your installation.
+  # Any of the DEFAULT_* environment variables can be removed to only create certain entities as required.
+  #  init:
```

Comment:
I don't at all love the idea of shipping commented code for the users to uncomment to enable something... just yuck.

Reply:
I cant really think of a neater way of doing this. It's also pretty common in docker-compose files to uncomment snippets to enable and disable services.